### PR TITLE
Add Tower Inventory module

### PR DESCRIPTION
**Pull request by wwitzel3**
##### ISSUE TYPE
 - New Module Pull Request

##### COMPONENT NAME
Ansible Tower inventory module

##### ANSIBLE VERSION

```
ansible 2.3.0 (tower_module_inventory f3c2dc08d6) last updated 2017/02/15 12:39:16 (GMT -400)
  config file =
  configured module search path = Default w/o overrides
```

##### SUMMARY
Allows playbook authors to create Ansible Tower inventories.

### CODE REVIEW

**Comment by ghjm** [comment]
shipit

**Comment by matburt** [comment]
shipit

**Comment by gundalow** [comment]
Merged and added into CHANGELOG https://github.com/ansible/ansible/commit/3b9892b45bab62d35547542f810371196f1dae82

File: lib/ansible/modules/web_infrastructure/ansible_tower/tower_inventory.py [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+        - The description to use for the inventory.
+      required: False
+      default: null
+    organization:
+      description:
+        - Organization the inventory belongs to.
+      required: True
+    variables:
+      description:
+        - Inventory variables. Use '@' to get from file.
```
Comment (bcoca) [inline lib/ansible/modules/web_infrastructure/ansible_tower/tower_inventory.py:47]: this is a bad pattern for a module, change to detect dict or string (assume path)

Comment (wwitzel3) [reply lib/ansible/modules/web_infrastructure/ansible_tower/tower_inventory.py:47]: Got it, thanks. I will update modules that are using this.

**Pull request merged**